src/screens/Ticketing/Ticket/Component/InspectionSymbol.tsx

```diff
@@ -76,8 +76,7 @@ const InspectableContent = ({
       <ThemeText
         type="body__primary--bold"
         allowFontScaling={false}
-        style={styles.symbolZones}
-        color={themeColor}
+        color={shouldFill ? themeColor : undefined}
```

Comment:
Instead of undefined could it be the default theme colour?

Reply:
The `ThemeText` component gives the appropriate text color based on theme, as opposed to just using `Text`, and if necessary the color can be overridden with the `color` parameter. So I don't think it is necessary to specifically send in the default theme color here, as `ThemeText` falls back to using it when the color parameter is undefined.